src/components/Article.jsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState} from "react"
+import axios from "axios"
+import { useParams } from "react-router-dom"
+
+function Article (){
+    const article = useParams()
+    const articleChoice = article.article_id
+    
+    const [oneArticle, setOneArticle] = useState([])
+     useEffect(()=>{
+         axios.get(`https://northcoders-news-api-bjpy.onrender.com/api/articles/${articleChoice}`)
+         .then((article)=>{
+             setOneArticle(article.data.article)
+         })
+ },[articleChoice])
+ return (
+<button>
```

Comment:
This shouldn't be a button, this would confuse screen readers, if you want it to look like a button style wise you can achieve this with CSS